<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1" colspan="3">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_3"/>
        <widget class="QWidget" name="page_4"/>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Functionality:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="4">
       <widget class="QLineEdit" name="ri_le"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Expected Result:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QComboBox" name="valid_cb"/>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Valid TCs:</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="4">
       <widget class="QTextEdit" name="note_le"/>
      </item>
      <item row="17" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="type_cb">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Test case ID:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Test Description:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="author_line"/>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Related Issue:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="4">
       <widget class="QTextEdit" name="er_le"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Steps:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Automation:</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QTextEdit" name="testDes_le"/>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QComboBox" name="testSuite_cb"/>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLineEdit" name="func_le"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Author:</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QComboBox" name="autoAvai_cb"/>
      </item>
      <item row="1" column="3">
       <widget class="QDateEdit" name="dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Type:</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Note:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Testsuite:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="4">
       <widget class="QTextEdit" name="steps_le"/>
      </item>
      <item row="17" column="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
